activerecord/test/cases/adapters/mysql2/datetime_precision_quoting_test.rb

```diff
@@ -25,28 +25,25 @@ class Mysql2DatetimePrecisionQuotingTest < ActiveRecord::Mysql2TestCase
     end
   end
 
-  test "no microsecond precision for MariaDB lt 5.3.0" do
-    stub_version "5.2.9-MariaDB" do
-      assert_no_microsecond_precision
-    end
-  end
```

Comment:
MariaDB lt 5.3.0 is not supported
https://github.com/rails/rails/commit/fb6743acc58495a63bca54221da9f38ce8227d3a#diff-98ce5dc0cd01f8b0435ff6feac10d8a8L100-R100